xml2rfc/writers/text.py

```diff
@@ -1125,28 +1126,21 @@ def render_authors(self, e, width, **kwargs):
         for i, author in enumerate(authors):
             if i == len(authors) - 1 and len(authors) > 1:
                 buf.append('and ')
-            organization = author.find('organization')
-            initials, surname = short_author_name_parts(author)
-            if surname:
-                initials = initials or ''
-                if i == len(authors) - 1 and len(authors) > 1:
-                    # Last author is rendered in reverse
-                    if len(initials) > 0:
-                        buf.append(initials + ' ' + \
-                                     surname)
-                    else:
-                        buf.append(surname)
-                elif len(initials) > 0:
-                    buf.append(surname + ', ' + initials)
-                else:
-                    buf.append(surname)
-                if author.attrib.get('role', '') == 'editor':
-                    buf.append(', Ed.')
-            elif organization is not None and organization.text:
```

Comment:
So we no longer the organization if the author name is missing?  I think that is a regression.

Reply:
[`ref_author_name_last`](https://github.com/ietf-tools/xml2rfc/blob/6e64a1c667892a9600c2a772615c1be3df855936/xml2rfc/util/name.py#L87) and [` ref_author_name_last`](https://github.com/ietf-tools/xml2rfc/blob/6e64a1c667892a9600c2a772615c1be3df855936/xml2rfc/util/name.py#L112) adds the `organization` if `name` is missing, so no longer needed to include that check.
Both HTML and PDF renderings make use of the same functions.

Reply:
Oh, this is done un util/name.py (ref_author_name_first/last).

Reply:
Yeah.

Reply:
A differences that ref_.... uses short_org...., while the existing code used the full name of the org.

Reply:
It appears that was a difference between text and html.  That is now fixed!?

Reply:
I have asked the RPC to advise

Reply:
This seems to be about how an organization's name appears in a reference when that org is an author. I think short name (when present) is preferred for the following reasons:
  * readability of the reference itself (assuming the org is well known by the short name)
  * usage in new bibxml (e.g., reference.IEEE 802-1AX.2020.xml) includes the abbrev attribute and it's preferred
```<organization abbrev="IEEE">Institute of Electrical and Electronics Engineers</organization>```

Whether or not to use an abbreviated name for orgs in references is not included in current style guidance. Over time, either has been used. FWIW, the RPC often uses only the abbreviated name. For example: rfc9182.notprepped.xml contains
```
<author><organization>ISO</organization></author> for [ISO10589]
<author><organization>IEEE</organization></author> for [IEEE802.1AX]
```

Reply:
Thank you!
So it seems the PR as proposed supports this approach best, while harmonizing TXT and HTML/PDF renderings.